domain/build.gradle

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2016 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+description = 'GoCD Common Module'
+
+test {
+  exclude "**/*DockerTest*"
+}
+
+dependencies {
+  compile project(':db')
+  compile project(':config-api')
+  compile project(':commandline')
+  compile group: 'org.springframework', name: 'spring-orm', version: '3.1.3.RELEASE'
+  compile group: 'cglib', name: 'cglib-nodep', version: '2.2'
+  compile group: 'org.codehaus.jettison', name: 'jettison', version: '1.2'
+  compile group: 'net.sf.kxml', name: 'kxml2', version: '2.3.0'
+  compile group: 'net.sf.kxml', name: 'kxml2-min', version: '2.3.0'
```

Comment:
No, you are right. Since I was moving out stuff from common module, I started out by copying the dependencies from there, but later on missed cleaning it up.